### PR TITLE
fix the loss of digits after the decimal point

### PR DESCRIPTION
**Pull request by hanhan77**

### CODE REVIEW

**Comment by miraclesu** [comment]
Thanks @hanhan77! Could you add some tests for this bug?

File: entities/fraction_test.go [hunk 1]
```diff
@@ -190,3 +190,23 @@ func TestDivide(t *testing.T) {
 		}
 	}
 }
+
+func TestToSignificant(t *testing.T) {
+	var tests = []struct {
+		Input  [2]int64
+		Output [1]string
```
Comment (miraclesu) [inline entities/fraction_test.go:197]: Output string should be better

**Review by miraclesu — CHANGES_REQUESTED** [review]

File: entities/fraction_test.go (hunk 1 above)
Comment (emma-chen-sz) [reply entities/fraction_test.go:197]: ok

**Review by miraclesu — APPROVED** [review]
LGTM. Thanks！

**Pull request merged**